presentation/common/src/presentation-common/PresentationRpcInterface.ts

```diff
@@ -171,7 +171,7 @@ export type ComputeSelectionRpcRequestOptions = PresentationRpcRequestOptions<Co
  * Interface used for communication between Presentation backend and frontend.
  * @public
  */
-export class PresentationRpcInterface extends RpcInterface {
+export class PresentationRpcInterface extends RpcInterface { // eslint-disable-line deprecation/deprecation
```

Comment:
I suppose this means `PresentationRpcInterface` should also be deprecated with the same comment?

Reply:
Not necessarily...

Unless we switch to a completely different paradigm on 5.0 (like REST), we will still have the concept of an “RPC interface” going forward.

I don’t see the need to deprecate this interface (or the imodel read or tile interfaces, etc) unless their own apis (ie, the set of remote apis that they describe) will be changing...

Reply:
I'm worried `RpcInterface` gets removed or completely changed, which would require the same for `PresentationRpcInterface`. That would force us to break our promise of not breaking APIs for 2 versions since deprecation.